src/main/java/core/basesyntax/FruitTransaction.java

```diff
@@ -0,0 +1,70 @@
+package core.basesyntax;
+
+import java.util.Arrays;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public String getFruit() {
+        return fruit;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public FruitTransaction setOperation(Operation operation) {
+        this.operation = operation;
+        return this;
+    }
+
+    public FruitTransaction setOperation(String operation) {
```

Comment:
setters should not return anything

Reply:
it's logic not for setters, you can search value in other place, and give object Operation in setOperation